apis/components/v1/dashboard_types.go

```diff
@@ -26,6 +26,17 @@ const (
 	DashboardInstanceName = "default-dashboard"
 )
 
+// +kubebuilder:rbac:groups=components.opendatahub.io,resources=dashboards,verbs=get;list;watch;create;update;patch;delete
```

Comment:
I don't think those annotation should be here, they should be on the controller as i fact, the need i.e. to have an RBAC for the `user.openshift.io` is a controller requirement, not an API one

Reply:
i moved these back to the kubebuilder_rbac.go for now, till we come up with a better idea